### PR TITLE
Install schematics if it doesn't exist

### PR DESCRIPTION
**Pull request by dxvladislavvolkov**

### CODE REVIEW

File: utility/run-command.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const spawn = require('child_process').spawn;
 
-module.exports = function(args, customConfig) {
-    const command = /^win/.test(process.platform) ? 'npx.cmd' : 'npx';
+module.exports = function(packageManagerName, args, customConfig) {
```
Comment (nightskylark) [inline utility/run-command.js]: Why `packageManagerName` if utility named `run-command`? Should not it be more abstract?

File: commands/application.angular.js [hunk 2]
```diff
@@ -18,20 +24,20 @@ function runSchematicCommand(schematicCommand, options, evaluatingOptions) {
...
+    runCommand('npx', commandArguments, evaluatingOptions);
 }
 
 const install = (options) => {
     runSchematicCommand('install', options);
 };
 
 const create = (appName, options) => {
-    runNpxCommand([
+    runCommand([
```
Comment (nightskylark) [inline commands/application.angular.js]: Did you forget the first argument?

File: commands/application.angular.js [hunk 3]
```diff
@@ -1,10 +1,16 @@
 const path = require('path');
-const runNpxCommand = require('../utility/run-npx-command');
+const fs = require('fs');
+const runCommand = require('../utility/run-command');
 
 function runSchematicCommand(schematicCommand, options, evaluatingOptions) {
     const collectionName = 'devextreme-schematics';
+    const packageJsonPath = path.join(process.cwd(), `node_modules/${collectionName}/package.json`)
```
Comment (nightskylark) [inline commands/application.angular.js]: The variable name does not tell `package.json` file of which package you are checking.

File: commands/application.angular.js [hunk 4]
```diff
@@ -1,10 +1,16 @@
...
+const fs = require('fs');
+const runCommand = require('../utility/run-command');
 
 function runSchematicCommand(schematicCommand, options, evaluatingOptions) {
     const collectionName = 'devextreme-schematics';
+    const collectionPackageJsonPath = path.join(process.cwd(), `node_modules/${collectionName}/package.json`)
     let collectionPath = collectionName;
 
+    if(!fs.existsSync(collectionPackageJsonPath)) {
+        runCommand('npm', ['install', collectionName]);
```
Comment (nightskylark) [inline commands/application.angular.js]: Should not we evaluate the next command only after this command will be finished?

Comment (dxvladislavvolkov) [reply commands/application.angular.js]: Commands will be executed sequentially. But I don't know how it works on different platforms. I think it's better to fix it.

**Pull request closed without merging**